src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -412,12 +413,14 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
 
         const auto objects = ObjectManager::getAvailableObjects(objectType);
         _tabObjectList.reserve(objects.size());
-        for (auto [index, object] : objects)
+        for (auto& [index, object] : objects)
         {
             auto entry = TabObjectEntry{ index, object, Visibility::shown };
             _tabObjectList.emplace_back(std::move(entry));
         }
 
+        std::sort(_tabObjectList.begin(), _tabObjectList.end(), [](const TabObjectEntry& lhs, const TabObjectEntry& rhs) { return lhs.object._name < rhs.object._name; });
```

Comment:
Might be a good time to start using std::ranges, sorting by member is very easy with it.